src/views/MainPage/components/Activity/Card/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { TextColorType } from '@src/lib/types/main';
+import * as S from './style';
+
+interface CardProps {
+  img: string;
+  navKor: string;
+  navEng: string;
+  description: TextColorType[];
+}
+
+export default function Card({ img, navKor, navEng, description }: CardProps) {
+  const blurMotion = {
+    rest: {
+      display: 'none',
+    },
+    hover: {
+      display: 'flex',
+    },
+  };
```

Comment:
1. 여기 display로 하신 이유가 있을까요?
2. 기왕 프레이머 모션 쓰는 김에 [여기](https://toss.im/career) 몰입의 힘을 믿습니다 부분처럼 뜨게 하는 건 어떨까요? 

```suggestion
  const blurMotion = {
    rest: { opacity: 0 },
    hover: { opacity: 1, transition: { duration: 0.5 } },
  };

  const textMotion = {
    rest: { y: 5 },
    hover: { y: 0, transition: { duration: 0.5 } },
  };
```

content들은 `<motion.div variants={textMotion}>` 이런식으로 감싸주고요

Reply:
이런 느낌

Uploading 화면 기록 2023-12-05 오후 4.56.35.mov…

Reply:
이부분은 따로 애니메이션에 대한 transition에 대한 피드백은 없어서 바로 마우스를 호버하면 바로 보이도록 하게 하였는데, 제안해준 코드가 훨씬 부드러운 것 같네요. 
이 부분은 디자이너한테 다시 물어보고 이후에 모바일일때 캐러셀로 보여주는 작업과 함께 진행하겠습니다!